### DISCUSSION TITLE
Why does `eza` target the nightly toolchain?

### DISCUSSION THREAD

**Discussion by daviessm** [eza-community/eza#170] [Development]
Just wondering if there's a reason that `rust-toolchain.toml` specifies `nightly` rather than a stable released version. I was surprised when `cargo` started downloading a new toolchain when building. `Cargo.toml` specifies `1.70.0` so shouldn't we use that?

I'm hoping to be able to package for Gentoo and that will require usage of `stable`.

**Comment by cafkafk** [comment]
There is no good reason, I'll open an issue so I don't forget.

**Comment by cafkafk** [comment]
Aaaaand it no longer does. #184

**Reply by daviessm** [reply]
Awesome, thanks!
